libraries/chain/asset_evaluator.cpp

```diff
@@ -410,11 +410,22 @@ void_result asset_update_bitasset_evaluator::do_evaluate(const asset_update_bita
  */
 static void update_bitasset_object_options(const asset_update_bitasset_operation& op, database& db, asset_bitasset_data_object& bdo, const asset_object& asset_to_update )
 {
-   // If the minimum number of feeds to calculate a median has changed, we need to recalculate the median
+   // If the minimum number of feeds to calculate a median has changed
+   // we need to recalculate the median
    bool should_update_feeds = false;
-   if( op.new_options.minimum_feeds != bdo.options.minimum_feeds )
+   if( op.new_options.minimum_feeds != bdo.options.minimum_feeds)
       should_update_feeds = true;
 
+   // We need to call check_call_orders if the settlement price changes after hardfork 890
+   bool after_hardfork_890 = false;
+   if (db.get_dynamic_global_properties().next_maintenance_time > HARDFORK_CORE_890_TIME) {
```

Comment:
`db.get_dynamic_global_properties()` is called twice in this function. We can use a local variable.

Reply:
Both were attempting to get the next maintenance time, so I'm throwing that in a local variable. If we need another value from dynamic_global_properties, we can throw it in a local as well.